bigint/bigint.mbti

```diff
@@ -0,0 +1,41 @@
+package moonbitlang/core/bigint
+
+// Values
+fn from_decimal_string(String) -> BigInt
+
+fn from_int(Int) -> BigInt
+
+fn from_int64(Int64) -> BigInt
+
+fn make_zero() -> BigInt
+
```

Comment:
TODO: we will add literal syntax support, e.g, `0n`

Reply:
annotated